README.md

```diff
@@ -177,6 +178,21 @@ td_agent_source 'test_in_tail' do
 end
 ```
 
+Use attribute `_raw_options` for attribute that using value such as `Array` or `Hash`. For example when you using [FluentD systemd plugin](https://github.com/reevoo/fluent-plugin-systemd).
+Just add the attribute name as an array item of `_raw_options`
+
+```ruby
+td_agent_source 'tail_journalctl' do
+  type 'systemd'
+  tag 'journalctl'
+  parameters(
+    matches: [{"_SYSTEMD_UNIT": "syslog.service"}]
```

Comment:
Honestly speaking, I'm skeptical if this is so useful. If I understand the situation correctly, just passing the parameter as a `String` must be sufficient for that purpose.

```rb
td_agent_source 'tail_journalctl' do
  ( ... )
  parameters(
    matches: ::JSON.generate([{"_SYSTEMD_UNIT": "syslog.service"}])
    read_from_head: true
  )
end
```

I know the `TdAgent::Helpers.params_to_text` isn't well implemented, but this change would make the situation more complicated with very few actual benefit, I think 🤔

Reply:
Hi, @yyuu thanks for reviewing my PR. I will try your approach and will tell you soon if this works on my case.

Reply:
Hi, @yyuu so I've been thinking about your approach. But I found a problem when using this method. How can user set this attributes from `nodes.json` files where we can't use ruby code? This will be okay if the user set the attribute directly in `attributes` files.

Thanks in advance.